Sources/TuistGenerator/Generator/ProjectDescriptorGenerator.swift

```diff
@@ -301,11 +311,8 @@ final class ProjectDescriptorGenerator: ProjectDescriptorGenerating {
         return attributes
     }
 
-    private func determineProjectConstants(graphTraverser: GraphTraversing) throws -> ProjectConstants {
-        if graphTraverser.hasPackages {
-            return .xcode11
-        } else {
-            return .xcode10
-        }
+    private func determineProjectConstants() throws -> ProjectConstants {
+        // TODO: Determine if this can be inferred by the set Xcode version
```

Comment:
Unsure how we want to handle this here. At this point we probably want to bump this anyway (Xcode 14.1 is the minimum version supported by Apple now) but I'm not sure if there's a smarter way to read this.

Reply:
I think we have something in Config.swift? However, I would just go for xcode 13 given it's already behind the minimum supported one

Reply:
We do for supported Xcode versions but not necessarily supported Xcode project file format version. 13 is definitely safe.

Reply:
Yeah but we can take the min of that and use it as Xcode project version I guess